src/main.js

```diff
@@ -20,17 +28,100 @@ function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
 
-function displayNewCover() {
+function displayNewCover() {  
   bookCover.src = covers[getRandomIndex(covers)];
   bookTitle.innerText = titles[getRandomIndex(titles)];
   firstDescriptor.innerText = descriptors[getRandomIndex(descriptors)];
   secondDescriptor.innerText = descriptors[getRandomIndex(descriptors)];
+  currentCover = new Cover(bookCover.src, bookTitle.innerText, firstDescriptor.innerText, secondDescriptor.innerText);
+}
```

Comment:
Wow, figuring out how to integrate this currentCover variable into our displayNewCover function took a long time!